storage-proofs-zexe/src/circuit/test/mod.rs

```diff
@@ -193,13 +193,13 @@ impl<E: Engine> TestConstraintSystem<E> {
         write!(s, "\n\n").unwrap();
 
         let negone = {
-            let mut tmp = E::Fr::one();
+            let tmp = E::Fr::one();
             // tmp.negate();
             -tmp
         };
 
         let powers_of_two = (0..  <E::Fr as PrimeField>::Params::MODULUS_BITS)
-            .map(|i| (E::Fr::from_repr_raw(<E::Fr as PrimeField>::BigInt::from(2 as u64))))
+            .map(|_| (E::Fr::from_repr_raw(<E::Fr as PrimeField>::BigInt::from(2 as u64))))
```

Comment:
Isn't this always equal to two?

Reply:
good catch, I think we missed this during our call. Fixed it!